float64[] tempo
string[] right_arm_motions
string[] left_arm_motions